internal/codegen/databasecodegen/readtransaction.go

```diff
@@ -246,13 +251,14 @@ func (g ReadTransactionCodeGenerator) generateListQueryStruct(f *codegen.File, t
 
 func (g ReadTransactionCodeGenerator) generateListMethod(f *codegen.File, table *spanddl.Table) {
 	const (
-		limitParam  = "limit"
-		offsetParam = "offset"
+		limitParam  = "__limit"
+		offsetParam = "__offset"
```

Comment:
nice! are there any limitations on parameter names in Spanner?

Reply:
Not that I can find - since it works with the emulator I think it should be fine!